### ISSUE TITLE
"500 - Internal Server Error" while downloading dataset

### ISSUE THREAD

**Issue by rajamanthan** [Kaggle/kaggle-api#83]
When I execute `kaggle datasets list -s hacker` command, it shows the list of datasets having the word `hacker` in it.

But, the following command is giving error -

`kaggle datasets download hacker-news/hacker-news`

This command is giving me `500 - Internal Server Error`. Please help!

**Comment by rajamanthan** [comment]
I tried downloading `zillow/zecon` dataset. It downloaded 100% successfully.

**Comment by rysteboe** [comment]
Hmm, looks like #68 wasn't fully fixed.  Investigating.

Sorry for delay - was at an off-site last week.

**Comment by rysteboe** [comment]
Ah whoops, nevermind.  This is (sorta) working as intended - you cannot download BigQuery datasets.  Will see if I can make the error message not terrible, but that may wind up being a surprising amount of work.

**Comment by rajamanthan** [comment]
No worries. Can we explicitly mention in the readme that "the api currently doesn't provide any kind of support for BigQuery type of datasets"? I am assuming that the api supports all other types of datasets.

**Comment by rysteboe** [comment]
Yeah, I'll add to the readme as well as fix the error message to be not awful.  Thanks!

**Comment by rajamanthan** [comment]
That would be great! Thank you! :)

**Comment by Ignacio-Ibarra** [comment]
Same error with 

```
import os
os.environ["KAGGLE_USERNAME"] = xxxxx
os.environ["KAGGLE_KEY"] = xxxxxxx

!kaggle datasets download uba-dmeyf2021-primera
```
Any help?

**Comment by erdalsivri** [comment]
> Same error with
> 
> ```
> import os
> os.environ["KAGGLE_USERNAME"] = xxxxx
> os.environ["KAGGLE_KEY"] = xxxxxxx
> 
> !kaggle datasets download uba-dmeyf2021-primera
> ```
> 
> 500 - Internal Server Error
> 
> Any help?

That's a competitions dataset so you need to use the `competitions` command:

```bash
$ kaggle competitions download -c uba-dmeyf2021-primera
```

If you get a Forbidden error, make sure you've accepted the rules of the competition: https://www.kaggle.com/c/uba-dmeyf2021-primera/data
